test/views/templates/payments/WhatYouOweChargeHelperSpec.scala

```diff
@@ -44,6 +51,16 @@ class WhatYouOweChargeHelperSpec extends ViewBaseSpec {
       }
     }
 
+    "payment should have a no to and from date to form the description, but doesn't" should {
+
+      val model = paymentModelNoPeriod(ReturnDebitCharge)
+      val helper = new WhatYouOweChargeHelper(model, Some(false), messages)
+
+      "omit the description of the payment" in {
+        helper.description shouldBe None
+      }
+    }
+
     "payment has a no to and from date" should {
```

Comment:
I know you didn't update this, but could you correct it please so it makes sense?